packages/lib-utils/src/app/redux/actions/k8s.ts

```diff
@@ -62,7 +63,25 @@ const WS: { [id: string]: WebSocketFactory } = {};
 const POLLs: { [id: string]: number } = {};
 const REF_COUNTS: { [id: string]: number } = {};
 
-const paginationLimit = 250;
+const PAGINATION_LIMIT = 250;
+const WS_TIMEOUT = 60 * 1000;
+
+type EventType = { object: K8sResourceCommon };
+
+const isOfEventType = (e: MessageDataType): e is EventType => {
+  return (e as EventType).object !== undefined;
+};
+
+type EventsType = K8sEvent[] & K8sResourceCommon[];
+
+const isOfEventsType = (events: MessageDataType[]): events is EventsType => {
+  const event = (events as EventsType)[0];
```

Comment:
@vidyanambiar do we ever expect there to be more than one event? I wonder if we should future proof this function by using `_.every` or similar?

Reply:
@florkbr That's a good point. I agree we should use `_.every` here. I pushed up an update 👍